src/core/dnd/types.ts

```diff
@@ -5,11 +5,14 @@
  * https://github.com/eclipsesource/jsonforms-editor/blob/master/LICENSE
  * ---------------------------------------------------------------------
  */
+import { UISchemaElement } from '@jsonforms/core';
```

Comment:
I think we should use the `LinkedUISchemaElement` from `../model`
@sdirix what do you think?

Reply:
yes make sense. I would use `LinkedUISchemaElement` wherever possible and only `UISchemaElement` where we have to because of JSON Forms.